static/app/views/organizationGroupDetails/allEventsTable.tsx

```diff
@@ -99,4 +65,88 @@ const AllEventsTable = (props: Props) => {
   );
 };
 
+type ColumnInfo = {columnTitles: string[]; fields: string[]};
+
+const getColumns = (group: Group, organization: Organization): ColumnInfo => {
+  const isPerfIssue = group.issueCategory === IssueCategory.PERFORMANCE;
+  const isReplayEnabled = organization.features.includes('session-replay-ui');
+
+  const {fields: platformSpecificFields, columnTitles: platformSpecificColumnTitles} =
+    getPlatformColumns(group.project.platform ?? group.platform, {isReplayEnabled});
+
+  const fields: string[] = [
+    'id',
+    'transaction',
+    'title',
+    'release',
+    'environment',
+    'user.display',
+    'device',
+    'os',
+    ...platformSpecificFields,
+    ...(isPerfIssue ? ['transaction.duration'] : []),
+    'timestamp',
+  ];
+
+  const columnTitles: string[] = [
+    t('event id'),
+    t('transaction'),
+    t('title'),
+    t('release'),
+    t('environment'),
+    t('user'),
+    t('device'),
+    t('os'),
+    ...platformSpecificColumnTitles,
+    ...(isPerfIssue ? [t('total duration')] : []),
+    t('timestamp'),
+    t('minidump'),
```

Comment:
I don't see `minidump` as an entry in the `fields` array, is there some hidden logic somewhere assigning this?

Reply:
Yeah, minidump is a special field handled by the table itself

Reply:
I dug into the events table and I see that it's manually getting pushed in there, would we be able to push the title along with the field instead of assigning it here?

Reply:
Not without more changes, as fields get added to the event view and become part of the discover query, fetching minidumps is not through discover.

Reply:
Actually mb, I might have misunderstood what you meant. I'd want to push it here, in the `allEventsTable.tsx`